db/fns.md

```diff
@@ -2667,15 +2667,19 @@ Returns a count of rows that have been deleted.
   * `secret jsonb`
   * `etag uuid`
   * `quarantine_until timestamptz`
-* *Last defined on version*: 71
+  * `first_claim timestamptz`
+  * `last_date_active timestamptz`
+* *Last defined on version*: 87
 
 Get non-stopped workers filtered by the optional arguments,
 ordered by `worker_pool_id`, `worker_group`, and  `worker_id`.
 If the pagination arguments are both NULL, all rows are returned.
 Otherwise, page_size rows are returned at offset `page_offset`.
 The `quaratine_until` contains NULL or a date in the past if the
 worker is not quarantined, otherwise the date until which it is
-quaratined. `providers_filter_cond` and `providers_filter_value` used to
+quaratined. `first_claim` and `last_date_active` contains information
+known to the queue service about the worker.
```

Comment:
How is last date active defined? Last time any API request was made to the queue, or only a specific subset?

Reply:
This looks like he's just using what I had added previously https://github.com/taskcluster/taskcluster/pull/5365/files#diff-3dcdf963e193e0ce3968b21e9a477bf3cb166b7480385482d217b5d9773f3116R60-R66

>Updated each time a worker calls `queue.claimWork`, `queue.reclaimTask`, and `queue.declareWorker` for this task queue.

Reply:
Yes, this is handled by queue, whenever `claimWork` or `reclaimTask` calls happens, queue marks workers this way, to know that they were doing something at this period of time